metacatalog/ext/standards_export/extension.py

```diff
@@ -38,6 +45,16 @@ def wrapper_entry(self: Entry, config_dict: dict, template_path: str) -> ET.Elem
         # add wrapper to Entry model
         Entry.standards_export = wrapper_entry
 
+        # add function create_iso19115 to api.catalog
+        def wrapper_api(session: Session, id_or_uuid: Union[int, str], config_dict: dict, path: str = None, template_path: str = './schemas/iso19115/iso19115-2.j2'):
+            StandardsExportExtension.create_standard_metadata(session, id_or_uuid, config_dict, path, template_path)
```

Comment:
This line is causing the error. 
As you add this wrapper_api function to the catalog API, this is the function that is called and this one is not returning anything.